web/packages/test/scripts/deploy-ethereum.sh

```diff
@@ -60,6 +60,8 @@ start_lodestar() {
             --params.ALTAIR_FORK_EPOCH 0 \
             --params.BELLATRIX_FORK_EPOCH 0 \
             --params.CAPELLA_FORK_EPOCH 0 \
+            --params.DEPOSIT_CHAIN_ID 11155111 \
+            --params.DEPOSIT_NETWORK_ID 11155111 \
```

Comment:
Curious about what are these two parameters for or required?

Reply:
They are the only Lodestar config I could find related to the chain ID. I will double check if they are required. 😄